packages/terra-demographics-banner/src/_sharedObjects.jsx

```diff
@@ -5,8 +5,10 @@ import styles from './DemographicsBanner.module.scss';
 
 const cx = classNames.bind(styles);
 
-// eslint-disable-next-line react/prop-types
-const DemographicsBannerValue = ({ label, value, abbrTitle }) => {
+/* eslint-disable react/prop-types */
+const DemographicsBannerValue = ({
+  label, value, valueAriaHidden = false, abbrTitle,
```

Comment:
I guess this was like this before this change, but why don't we have these broken out as their own react components? These don't seem like `utils` to me.